mdadm.8.in

```diff
@@ -1878,11 +1878,15 @@ script.
 .SH For Monitor mode:
 .TP
 .BR \-m ", " \-\-mail
-Give a mail address to send alerts to.
+Give a mail address to send alerts to. Can be configured in
+.B mdadm.conf
+as MAILADDR.
 
 .TP
 .BR \-p ", " \-\-program ", " \-\-alert
-Give a program to be run whenever an event is detected.
+Give a program to be run whenever an event is detected. Can be configured in
```

Comment:
This doesn't really need to be a program. Maybe "program/command". 

This is just a thought, I'm fine either way.

Reply:
I don't want to make it complicated for now, let's leave it as is.